grafana-node-exporter-dashboard.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Node Exporter 面板重要监控指标详解
+aliases: ['/docs-cn/dev/grafana-node-exporter-dashboard/','/docs-cn/dev/reference/key-monitoring-metrics/node-exporter-dashboard/']
+---
+
+# Node Exporter 面板重要监控指标详解
+
+使用 TiUP 部署 TiDB 集群时，一键部署监控系统 (Prometheus & Grafana)，监控架构参见 [TiDB 监控框架概述](/tidb-monitoring-framework.md)。
+
+目前 Grafana Dashboard 整体分为 PD、TiDB、TiKV、Node_exporter、Overview 等。
+
+以下为 Node Exporter Dashboard 监控说明：
+
+## Overview
+
+- Virtual CPUs: CPU 核心数量
+- Total RAM: 内存总大小
+- Total Swap: 可使用的交换内存大小
+- System Uptime: 系统启动时长
+- Node_exporter version: Node Exporter 使用版本
+- CPU Used: CPU 的使用率
+- Memory Used: 内存的使用率
+- Swap Used: 交换内存的使用率
+- Load1: 1 分钟的负载情况
+- Load5: 5 分钟的负载情况
+- Load15: 15 分钟的负载情况
+
+## Kernel
+
+- OS: 操作系统类型
+- Machine: 机器平台
+- Kernel version: 内核版本
+- Interrupts: 内核中断情况
+- Forks: Linux 下创建进程的系统调用情况
+- Context Switches: 内核任务切换情况
+
+## CPU
+
+- CPU Usage: CPU 的使用率
+
+## Memory
+
+- Memory: 内存的使用情况
+- Distribution: 内存分布情况
+- Available Buddy Pages - Zone Normal: linuxbuddy 系统管理物理内存的 debug 信息，低值表示内存碎片严重
+- Active/Inactive: 活跃或非活跃内存情况，非活跃的内存将会优先回收，活跃的内存，除非绝对必要否则不会回收
+- Writeback and Dirty: 回写和脏页。回写：准备主动回写硬盘的缓存页；临时回写：用于临时写回缓冲区的内存；脏页：需要写回磁盘的数据大小
```

Comment:
Writeback %lu
                     Memory which is actively being written back to the
                     disk.